ada_feeding_actionselect/config/acquisition_library.yaml

```diff
@@ -0,0 +1,239 @@
+# Library of Acquisition Actions
+# Quats: W, X, Y, Z (matching Eigen)
+actions:
+    - # Default Action (VS90)
```

Comment:
Can you add a unique name/ID to each action here (and in the AcquisitionSchema message)? That'll make it easier if we ever need to check equality across actions (e.g., even if action select tells the robot to retry the same action, the robot could realize that it already tried that action and instead try the most likely unique action).

Reply:
I think this is too likely to introduce programmer error (i.e. having the same ID for different actions). I think an action is adequately identified by its unique vector (everything except for `pre_rot_hint`).

Reply:
And in the discrete case all actions are manually defined, so no need to worry about float rounding issues.

In the slightly more complicated case where actions are programmatically defined, it is much more likely that they are programmatically defined in advance and added to a YAML file before execution, so we still won't run into any float rounding issues.